bin/scripts/set_additional_api_configs.js

```diff
@@ -0,0 +1,38 @@
+var pluginManager = require('../../plugins/pluginManager.js'),
```

Comment:
Can we move this script to bin/upgrade/dev/scripts and change relative paths to that too